src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Constructors.jl

```diff
@@ -2,13 +2,26 @@
 # Lower case constructors
 ################################################################################
 
-projective_scheme(S::MPolyDecRing) = ProjectiveScheme(S)
+function proj(S::MPolyDecRing)
+  is_standard_graded(S) || error("ring must be standard graded")
+  return ProjectiveScheme(S)
```

Comment:
It seems to me that the better place for this input check would be the actual constructor
`ProjectiveScheme`? (however this need not delay merging this pull request.)